lms/djangoapps/completion_api/tests/test_views.py

```diff
@@ -146,3 +149,48 @@ def test_staff_access(self):
         self.client.force_authenticate(user)
         response = self.client.get('/api/completion/v0/course/?user=test_user')
         self.assertEqual(response.status_code, 200)
+
+
+@override_settings(STUDENT_GRADEBOOK=True)
+class CourseSummaryViewTestCase(SharedModuleStoreTestCase):
+    """
+    Test that the CourseSummaryView returns accurate summary data.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(CourseSummaryViewTestCase, cls).setUpClass()
+        cls.course = ToyCourseFactory.create()
+
+    def setUp(self):
+        super(CourseSummaryViewTestCase, self).setUp()
+        self.learners = [
+            UserFactory.create(username='learner%s' % i, email='learner%s@example.com' % i, password='test_pass')
```

Comment:
Nit: Can you just use `UserFactor.create()` or is it helpful to specify these values?